doc/RELEASE.md

```diff
@@ -0,0 +1,28 @@
+# Steps to release Mark Text
+
+- Create a release candidate
+  - Create branch `release-v%version%`
+  - Set environment variable `MARKTEXT_IS_OFFICIAL_RELEASE` to `1` (default on AppVeyor and Travis CI)
+  - Ensure [changelog](https://github.com/marktext/marktext/blob/master/.github/CHANGELOG.md) is up-to-date
+  - Bump version
+  - Update all `README.md` files
+  - Bump Flathub version ([marktext.appdata.xml](https://github.com/marktext/marktext/blob/master/resources/linux/marktext.appdata.xml))
+  - Create commit `release version %version%`
+  - Ensure all tests pass
+  - A new draft release should be available or create one
+- Publish GitHub release
+  - Add git tag `v%version%`
+  - Add changelog
+  - Add SHA256 checksums
```

Comment:
Sorry, I still didn't know how and where will us use the `SHA256` checksums? 😄

Reply:
It is intended to verify the downloaded file(s). The checksum is calculated by AppVeyor or Travis CI; it should be calculated before/after uploading the binaries.